scripts/run-dependencies.sh

```diff
@@ -1,9 +1,25 @@
 #!/usr/bin/env bash
-docker run --rm -d --network=host --name modelapi_redis redis
-docker run --rm -d --network=host --name modelapi_model mrcide/modelapi:latest
+set -ex
 
-docker run --rm -d --network=host --name hint-db mrcide/hint-db:mrc-371
+NETWORK=hint_nw
+CONTAINER=hint_db
+docker network create $NETWORK
+
+docker run --rm -d \
+  --network=$NETWORK \
+  --name $CONTAINER \
+  -p 5432:5432 \
+  mrcide/hint-db:mrc-371
+
+# From now on, if the user presses Ctrl+C we should teardown gracefully
```

Comment:
I don't think the trap needs to be here as well as in the dev script?

Reply:
Not super important but allows the script to be stopped any point from here on, rather than just at the end.